lang/lsp/src/hover.rs

```diff
@@ -41,15 +41,17 @@ fn ctx_to_markdown(ctx: &Ctx, value: &mut String) {
     value.push_str("**Context**\n\n");
     value.push_str("| | |\n");
     value.push_str("|-|-|\n");
-    for Binder { name, typ } in ctx.bound.iter().rev().flatten() {
-        if name == "_" {
-            continue;
+    for binder in ctx.bound.iter().rev().flatten() {
+        match binder {
+            Binder::Var { name, typ } => {
+                value.push_str("| ");
+                value.push_str(name);
+                value.push_str(" | `");
+                value.push_str(typ);
+                value.push_str("` |\n");
+            }
+            Binder::Wildcard { .. } => continue,
```

Comment:
Taking inspiration from Lean, we may still want to display them. In particular, we may want to display wildcards together with their De Bruijn index (or some other way to disambiguate them) s.t. one can distinguish multiple wildcards on the type level (e.g. `MyType(x, _@0.1, _@0.2)` rather than  `MyType(x, _, _)`.